CHANGELOG.md

```diff
@@ -7,23 +7,29 @@ The public API of this library consists of the functions declared in file
 
 ## [Unreleased]
 
+## [4.0.0-rc1] - 2022-02-02
+### Breaking changes
+- Rename functions according to the terminology RFC (#571, #403, #466, #495, #466)
+- Functions that can experience errors now have an `H3Error` return value. (#551, #550, #509, #505, #507, #508, #503, #478, #468, #436, #359)
+- Cell count parameters use `int64_t` instead of `int`.
+- `polygonToCells` (previously `polyfill`) accepts a flags argument for future expansion. (#570)
+
 ### Added
-- Vertex mode and associated functions:
+- Vertex mode and associated functions: (#422, #420, #417)
     - `cellToVertex(cell, vertexNum)`
     - `cellToVertexes(cell, out)`
     - `vertexToPoint(vertex, out)` 
```

Comment:
If this is still the name, it's a rename candidate - should be `vertexToLatLng` now

Reply:
It's actually been renamed to vertexToLatLng, changelog should be updated